core/store/migrations/migration0/migrate.go

```diff
@@ -98,22 +100,22 @@ type TxAttempt struct {
 // TaskRun stores the Task and represents the status of the
 // Task to be ran.
 type TaskRun struct {
-	ID                   string    `json:"id" gorm:"primary_key;not null"`
-	JobRunID             string    `json:"-" gorm:"index;not null;type:varchar(36) REFERENCES job_runs(id) ON DELETE CASCADE"`
-	ResultID             uint      `json:"-"`
-	Status               string    `json:"status"`
-	TaskSpecID           uint      `json:"-" gorm:"index;not null REFERENCES task_specs(id)"`
-	MinimumConfirmations uint64    `json:"minimumConfirmations"`
-	CreatedAt            time.Time `json:"-" gorm:"index"`
+	ID                   string `gorm:"primary_key;not null"`
+	JobRunID             string `gorm:"index;not null;type:varchar(36) REFERENCES job_runs(id) ON DELETE CASCADE"`
```

Comment:
Should this have `type:uuid`? Or is this an issue with the discrepancy between sqlite and postgres?